devlake-go/api/sql_client/sql_queries/monthly_mltc.sql

```diff
@@ -0,0 +1,45 @@
+-- Metric 2: median change lead time per month
+with _pr_stats as (
+-- get the cycle time of PRs deployed by the deployments finished each month
+    SELECT
+        distinct pr.id,
+        date_format(cdc.finished_date,'%y/%m') as month,
+        ppm.pr_cycle_time
+    FROM
+        pull_requests pr
+        join project_pr_metrics ppm on ppm.id = pr.id
+        join project_mapping pm on pr.base_repo_id = pm.row_id and pm.`table` = 'repos'
+        join cicd_deployment_commits cdc on ppm.deployment_commit_id = cdc.id
+    WHERE
+        (
+                    :project = ""
+                    OR LOWER(repos.name) LIKE CONCAT('%/', LOWER(:project))
```

Comment:
I think we need one more join on the `repos` table to filter on the repo name. When I run the API I get: `Error 1054 (42S22): Unknown column 'repos.name' in 'where clause'`